mmdet/models/necks/pafpn.py

```diff
@@ -0,0 +1,129 @@
+import torch.nn as nn
+import torch.nn.functional as F
+
+from mmdet.core import auto_fp16
+from mmdet.ops import ConvModule
+from ..registry import NECKS
+from .fpn import FPN
+
+
+@NECKS.register_module
+class PAFPN(FPN):
+    """Path Aggregation Network for Instance Segmentation.
+
+    This is an implementation of the PAFPN in Path Aggregation Network
+    (https://arxiv.org/abs/1803.01534)
+
+    Args:
+        in_channels (List[int]): number of input channels per scale
+
```

Comment:
- No need to add a blank line here.
- Use upper case for the first character, e.g., Number.